Sources/SwiftDocC/Infrastructure/Link Resolution/ExternalPathHierarchyResolver.swift

```diff
@@ -163,14 +161,14 @@ final class ExternalPathHierarchyResolver {
         }
     }
     
-    convenience init(dependencyArchive: URL) throws {
+    convenience init(dependencyArchive: URL, fileManager: FileManagerProtocol) throws {
```

Comment:
Should we have a default value here ` = FileManager.default` to avoid having to call this from each call site?

Reply:
This is only called in one place. I would prefer to force the caller to do the right thing and think about how it accesses files on disk